tests/fixtures/Item.php

```diff
@@ -26,7 +26,7 @@ public function getContent() {
 	 *
 	 * @param mixed $content        	
 	 */
-	public function setContent($content) {
+	public function setContent($content): self {
```

Comment:
What about adding the string type hint for `$content`.

Reply:
Done